vignettes/Trelliscope_Vignette.Rmd

```diff
@@ -84,7 +84,7 @@ Let's start with the simplest input data to run these trelliscope functions,
 e_data. For context, a typical e_data file looks like this: 
 
 ```{r, echo = F}
-datatable(head(pmartRdata::lipid_edata_pos), options = list(`scrollX` = T))
+datatable(head(pmartRdata::lipid_pos_edata), options = list(`scrollX` = T))
```

Comment:
fails since the DT library is not a dependency I think.  I think we can change datatable -> knitr::kable()

Reply:
Good idea! Done.